lib/esignatur/user.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'esignatur/api_resource'
+
+module Esignatur
+  # esignatur user representation
+  # More info: https://api.esignatur.dk/Documentation/User
+  class User
+    include ApiResource
+
+    DEFAULT_BASE_URL = 'https://api.esignatur.dk'
+
+    def initialize(api_key:, base_url: DEFAULT_BASE_URL)
+      @api = Esignatur::Api.new(api_key: api_key, base_url: base_url)
+      @attributes = {}
+    end
+
+    def find(user_id:, creator_id:)
```

Comment:
user instance should represent user, but `find` method modifies user and you can get results for different users in same instance. I find it very confusing. My suggestion - make `find` class method. Also it would be nice to mimic ActiveRecord dsl, so I would recommend to use `find_by` name instead of `find`